evaluation/evaluate.py

```diff
@@ -140,7 +141,7 @@ def run_evaluation(
 
     with open(results_dir / "evaluate_parameters.json", "w", encoding="utf-8") as parameters_file:
         parameters = {
-            "evaluation_gpt_model": openai_config.get("model", "unknown_model"),
+            "evaluation_gpt_model": openai_config.model,
```

Comment:
> Why?

Because it showed an error of "AttributeError: 'AzureOpenAIModelConfiguration' object has no attribute 'get'"

Reply:
Makes sense, doesn't it, because you've changed the `openai_config` from a `dict` (which supports "get") to a `ModelConfiguration` object. I've asked elsewhere why that decision was made.